### PR TITLE
Prevent empty keywords from being saved when using KeywordModelForm

### LINKED ISSUE #2701: Blank Keywords
**Describe the bug**
A journal or article can have a blank keyword.

**Janeway version**
1.4.0

**Expected behavior**
We should now allow blank keywords to be added.

### PR DESCRIPTION
**Pull request by mauromsl**
closes #2701

### CODE REVIEW

File: src/utils/tests.py [hunk 1]
```diff
@@ -145,3 +152,34 @@ class Meta:
...
+                model = journal_models.Journal
+                fields = ('keywords', )
+                exclude = tuple()
+        expected = "Expected Keyword"
+        data = {
+            "keywords": "Keyword, another one, and another one, %s" % expected
+        }
+        form = KeywordTestForm(data, instance=self.journal)
+        journal = form.save()
+        self.assertTrue(journal.keywords.filter(word=expected).exists())
```
Comment (joemull) [inline src/utils/tests.py:169]: This test has the same name as the one below--does it need a unique name?

**Review by joemull — CHANGES_REQUESTED** [review]
Just a question about the test names.

**Review by ajrbyers — APPROVED** [review]

**Comment by ajrbyers** [comment]
@mauromsl looks like tests are failing on jenkins.

**Comment by ajrbyers** [comment]
test this chuck norris

**Comment by ajrbyers** [comment]
test this please

**Pull request merged**